python/ray/data/_internal/execution/interfaces/physical_operator.py

```diff
@@ -459,3 +459,8 @@ def implements_accurate_memory_accounting(self) -> bool:
     def supports_fusion(self) -> bool:
         """Returns ```True``` if this operator can be fused with other operators."""
         return False
+
+    @property
+    def estimated_output_num_rows(self) -> Optional[int]:
```

Comment:
nit: rename property to keep naming convention consistent with `num_outputs_total`
```suggestion
    def num_output_rows_total(self) -> Optional[int]:
```

Reply:
also, can we add `_estimated_output_num_rows` as a class attribute in `PhysicalOperator.__init__()` (set to `None`)? so that here, we can simply return `self._estimated_output_num_rows`. and in `MapOperator`, we won't need to define its own `estimated_output_num_rows`